department-of-veterans-affairs/caseflow
# Khalin/appeals 53951

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-53951)

# Description
Please explain the changes you made here.

## Acceptance Criteria
As an Inbound Ops Team Superuser or Inbound Ops Team Supervisor you now have the ability to add tasks not related to an appeal on a piece of correspondence in a Correspondence Detail Page, so that I can track and report correspondence. 

## Testing Plan

1. Log into Caseflow as an Inbound Ops Supervisor user
2. Go to "Your Queue"
3. Click "Correspondence Cases" in the "Switch Views" drop down
4. Click on the "Pending" Tab
5. Click any veteran who still has tasks not related to appeal to be added(You may need to repeat this step until you find one).
6. Scroll to the bottom and click the "+" next to the "Tasks not related to an appeal" section
7. Click the "+ Add task" button and you will see a modal pop up titled "Add New Task"
8. Select any task and type "Test" into the text box
9. Click "Next"(Things may take some time to load)
10. Once loading is complete the modal will close and you will see the task you added in the task list within the "Tasks not related to an appeal" section along with a confirmation banner. 
11. Repeat steps 1-10 as an Inbound Ops Superuser
